webserver/src/main/java/io/kestra/webserver/services/ai/gemini/GeminiAiService.java

```diff
@@ -0,0 +1,362 @@
+                      If nothing is mentioned regarding the change location or the scope, feel free to discard the initial Flow if it doesn't fit the user's need.
+                      If you can make any relation between what is existing and what the user's want, try to avoid duplicating intent (example, if a user say "say hello world" and it already has a Log task that says "hi", just replace the text "hi" with "hello world").
+                    - Use only the types and properties explicitly defined in the above schema. Do not invent, guess, or use properties from other types.
+                    - If a property is not present in the schema for a given type or you are unsure whether it exists or not DO NOT INCLUDE IT.
+                    - The type of each property must match the schema exactly.
+                    - Do not use any types not present in the schema in a given section.
+                    - When accessing variables within {{...}} expressions, you MUST ONLY use variables that are explicitly used in similar {{...}} expressions in the examples provided for that type or are explicitly documented as available for the specific task or context within the provided schema. Beware that some properties may not be available in the context of the task you are building.
+                    - There are a few global variables that are always available in the context of the Flow, such as `{{ flow.id }}`, `{{ flow.namespace }}`, `{{ flow.name }}`, `{{ flow.description }}`, `{{execution.id}}`, ... You can use them in your Pebble expressions but refer to kestra.io documentation to know if they exist.
+                    - Examples are very valuable and are fully working. Use them to guide property usage, structure, and Pebble expressions. However, don't copy them verbatim; adapt them to the current context, and if an example uses outputs, be careful about the type used in the example versus the one you are using.
+                    - If the examples are using a type, you can also use it, but you may only be able to change property values respecting the inferred value type (for example, Return task may be used for outputting index).
+                    - Some properties accept either a string (Pebble expression for example) or an array/object. If it's the case, you can leverage Pebble functions and filters. As an example, `{{ range(1, 101) }}` will generate an array of numbers from 1 to 100. Refer to kestra.io documentation to know the functions that exist.
+                    - Beware about `default` values of properties and if needed tweak them to avoid too long waiting times or if a better value could be used to suit user's intent.
+                    - Root-level outputs in the Flow are used to return values from the Flow execution, not display them. If the user asks explicitly that the Flow outputs something for another flow / execution to use it, you can include some, otherwise it's useless.
```

Comment:
```suggestion
                    - Flow-level outputs are used to return values from the Flow execution. If the user asks that the Flow should output something, you can include flow outputs. Otherwise, use only task outputs and only if explicitly requested to pass data between tasks.
```
confusing and too long. if the suggested option doesn't match what you wanted to say, can you explain? frankly, even knowing kestra, not sure what is meant by root-level outputs, task outputs or flow outputs or sth else?

Reply:
sometimes it's using flow outputs thinking it will expose things to tasks or display them directly

Reply:
got it, good point, clarified in the suggestion (I think flow outputs or flow-level outputs gives a bit clearer instruction than root-level outputs)